10up/simple-local-avatars
# upkeep/php-reqs: Update PHP compat check to prevent fatal error

### Description of the Change



Recently Publisher Media Kit reported a fatal issue caused due to redeclaration of a function across our 10up plugins.
Ref: https://github.com/10up/publisher-media-kit/issues/196

This PR fixes that by using [10up/wp-compat-validation-tool](https://github.com/10up/wp-compat-validation-tool).

### How to test the Change


Try running this PR on a PHP env lesser than the minimum supported version (7.4) and see the plugin gracefully exit with an error message in the admin screen.

### Changelog Entry


> Changed - Update PHP compatibility check by using [10up/wp-compat-validation-tool](https://github.com/10up/wp-compat-validation-tool)


### Credits

Props @Sidsector9 


### Checklist:
